pkgs/build-support/flutter/default.nix

```diff
@@ -115,7 +115,16 @@ in
 builtins.foldl'
   (prev: package:
   if packageOverrideRepository ? ${package.name}
-  then prev.overrideAttrs packageOverrideRepository.${package.name}
+  then
+    prev.overrideAttrs
+      (packageOverrideRepository.${package.name} {
+        inherit (package)
```

Comment:
The whole `package` attribute is actually inherited here, but I list the attribute names explicitly for a couple of reasons.

1. If the format of `dart pub deps --json` changes (which is unlikely as the `dart pub deps` output doesn't change much in general, and `--json` hasn't changed since its initial implementation in March 2021), evaluation will fail once in a consistent place
2. It makes the available metadata attributes clear